roles/kubernetes/control-plane/templates/kubeadm-config.v1beta4.yaml.j2

```diff
@@ -477,6 +477,42 @@ scheduler:
 {% endfor %}
 {% endif %}
 ---
+apiVersion: kubeadm.k8s.io/v1beta4
+kind: UpgradeConfiguration
+apply:
+  kubernetesVersion: v{{ kube_version }}
+  allowExperimentalUpgrades: true
```

Comment:
HI @chadswen 

Would it make sense to configure `allowExperimentalUpgrades` as a variable with a default value of false?

Reply:
Changing the default false would technically be a breaking change. Some people may depend on the current default where it's set to true to install rc/alpha/beta versions right now. However, as noted above they would also need to add the checksums for these releases to be able to install those versions. 

I think it's safe to leave as is, since anyone using alpha/beta/rc release has to be very intentional about it anyway by adding checksums.